openquake/hazardlib/imt.py

```diff
@@ -300,7 +300,7 @@ def LiqOccur():
 
 def LSE():
     """
-    Liquefaction spatial extent as percentage of a pixel area.
+    Liquefaction or Landslide spatial extent.
```

Comment:
On second thought, if the user is running a liquefaction analysis and a landslide analysis in the same job, the two spatial extent IMTs will need to be separated otherwise the second will overwrite the first. We might not be able to use the same IMT for both after all.

Reply:
Part of future enhancement